test/TestUtilities/Microsoft.Internal.NuGet.Testing.SignedPackages/Asn1/OcspSignature.cs

```diff
@@ -61,8 +61,11 @@ internal static OcspSignature Decode(AsnReader reader)
                 while (certsSequenceReader.HasData)
                 {
                     ReadOnlyMemory<byte> data = certsSequenceReader.ReadEncodedValue();
+#if NET9_0_OR_GREATER
```

Comment:
I tried bumping some stuff to .NET 9 a while back and could never get it to work for this same reason!